purchase/app/purchase/CheckoutShippingController.java

```diff
@@ -75,17 +75,9 @@ public CheckoutShippingController(final ControllerDependency controllerDependenc
     }
 
     private F.Promise<Cart> updateCart(final PlayJavaSphereClient sphere, final Cart cart, final CheckoutShippingFormData checkoutShippingFormData, final CheckoutShippingPageContent content) {
-        final Address shippingAddress = content.getShippingForm().getShippingAddress().toAddress();
-        final Address nullableBillingAddress = content.getShippingForm().isBillingAddressDifferentToBillingAddress()
-                ? content.getShippingForm().getBillingAddress().toAddress()
-                : null;
         final String shippingMethodId = checkoutShippingFormData.getShippingMethodId();
-        final List<UpdateAction<Cart>> updateActions = asList(
-                SetShippingAddress.of(shippingAddress),
-                SetBillingAddress.of(nullableBillingAddress),
-                SetShippingMethod.of(Reference.of(ShippingMethod.referenceTypeId(), shippingMethodId))
-        );
-        return sphere.execute(CartUpdateCommand.of(cart, updateActions));
+        final SetShippingMethod setShippingMethod = SetShippingMethod.of(Reference.of(ShippingMethod.referenceTypeId(), shippingMethodId));
```

Comment:
I think that would be really great, since it's a very common use case
https://github.com/sphereio/sphere-jvm-sdk/issues/920